cfme/tests/test_login.py

```diff
@@ -277,3 +126,56 @@ def test_multiregion_displayed_on_login(context, setup_multi_region_cluster, mul
         login_view = navigate_to(rapp.server, 'LoginScreen')
         assert login_view.is_displayed
         assert 'Remote' in login_view.details.region.text
+
+
+@pytest.mark.parametrize('context', [ViaUI])
+@pytest.mark.meta(blockers=[BZ(1632718)])
```

Comment:
This BZ is verified . It can be removed .
This can be added instead @pytest.mark.meta(automates=[BZ(1632718)])

Reply:
done